misc/python/materialize/parallel_workload/worker.py

```diff
@@ -88,7 +87,8 @@ def run(self, host: str, port: int, user: str, database: Database) -> None:
                         self.exe
                     )
                     self.exe.reconnect_next = False
-                action.run(self.exe)
+                if action.run(self.exe):
```

Comment:
This is the only use of the return value of `run()` right?

Reply:
Correct, it's just for counting the actions since I was afraid that sometimes an action might always not run for some reason, and previously we still counted it.